libs/prebuilt/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -595,7 +617,16 @@ class Agent,Tools otherClass
         if response_format is not None:
             required_keys.add("structured_response")
 
-        if missing_keys := required_keys - set(state_schema.__annotations__):
+        # Pydantic v2
```

Comment:
shouldn't we just be using get_type_hints for all types? That supports inherited fields I believe

Reply:
yup you're right, and it was in the original change too! will bring it back